openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -139,6 +144,19 @@ def _add_s3_bucket_data(self):
             f"{self.cluster_info['name']}-{self.cluster_info['shortuuid']}.zip"
         )
 
+    def check_and_assign_aws_cluster_region(self):
+        if self.cluster_info["platform"] in [AWS_STR, AWS_OSD_STR]:
+            if not self.cluster_info.get("region"):
+                region, vpcs = None, None
```

Comment:
can we do `region, vpcs = 0, 0` and then `if vpcs is None` is not needed anymore